k8s/nginx-https/container/nginx.conf.template

```diff
@@ -158,8 +158,8 @@ stream {
   # Enable logging when connections are being throttled.
   limit_conn_log_level notice;
   
-  # Allow 16 connections from the same IP address.
-  limit_conn two 16;
+  # Allow 64 connections from the same IP address.
```

Comment:
I am of the opinion that since the MongoDB connection pool size is 100, and we have a 3 node cluster, we should bump this up to something closer to 300. Then MongoDB can use the default configs and will not face socket errors.

This will be a mix of connections via NGINX and connections between BDB and MDB in the node.
My intuition says maybe something like 256.